src/Gui/EditorView.h

```diff
@@ -116,6 +116,7 @@ private Q_SLOTS:
 
 private:
     EditorViewP* d;
+    QList <QMetaObject::Connection> connection_lst;

```

Comment:
For consistency (and to ensure `lst` doesn't read as "first"):
```suggestion
    QList <QMetaObject::Connection> connectionList;
```